Formula/a/apitrace.rb

```diff
@@ -0,0 +1,133 @@
+class Apitrace < Formula
+  desc "Tools for tracing OpenGL, Direct3D, and other graphics APIs"
+  homepage "https://apitrace.github.io/"
+  url "https://github.com/apitrace/apitrace/archive/00706d1a395ab6570ab88136fffb3028a3ded3bd.zip"
+  version "11.1+dev.0"
+  sha256 "0b51d3d6e9db9b42fc47c3ffad5b9ae68054a2a0e518f4c8c287316842c04757"
+  license all_of: ["MIT", "BSD-3-Clause", "libpng-2.0", "Zlib"]
+
+  # Known problems:
+  # - libbacktrace only meant to be used for ELF support / only relevant for Linux
+  # - gtest required in build-system, but unused
```

Comment:
Problematic to fix: I'd need ugly patches for the upstream build system.
Instead, I will probably address this upstream sometime, but for now it's fine to install a useless library.

Or shall I force build-time tests (how)?